tests/sles4sap/qesapdeployment/destroy.pm

```diff
@@ -0,0 +1,27 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Test for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    qesap_execute(cmd => 'ansible', cmd_options => '-d', verbose => 1, timeout => 60 * 5);
```

Comment:
Same comment (and in the line below) for the multiplication operation of 2 constants.